docs/configuration.md

```diff
@@ -28,6 +28,16 @@ k0s supports providing only partial configurations. In case of partial configura
     sudo k0s start
     ```
 
+## Configuring k0s via k0sctl
+
+k0sctl can deploy your configuration options at cluster creation time. Your
+options should be placed in the `spec.k0s.config` section of the k0sctl's
```

Comment:
I guess that could be an anchor link to the readme section

Reply:
Yeah, considered that. But it felt a bit redundant with the general link to the README in the next sentence, which I didn't want to remove. WDYT?